modules/update-changing-update-server-web.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster.adoc
+
+[id="update-changing-update-server-web_{context}"]
+= Changing the update server by using the web console
+
+Changing the update server is optional. If you have an OpenShift Update Service (OSUS) installed and configured locally, you must set the URL for the server as the `upstream` to use the local server during updates. 
+
+.Procedure
+
+. Navigate to *Administration* -> *Cluster Settings*, click *version*.
+. Click the *YAML* tab and then edit the `upstream` parameter value:
++
+.Example output
++
+[source,yaml]
+----
+  ...
+  spec:
+    clusterID: db93436d-7b05-42cc-b856-43e11ad2d31a
+    upstream: '<update-server-url>' <1>
+  ...
+----
+<1> The `<update-server-url>` variable specifies the URL for the update server.
++
+The default `upstream` is \https://api.openshift.com/api/upgrades_info/v1/graph.
```

Comment:
Same request here, since \https://api.openshift.com/api/upgrades_info/v1/graph is an [example URL](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls), please add backticks:

`\https://api.openshift.com/api/upgrades_info/v1/graph`

Reply:
Done